### ISSUE TITLE
*** buffer overflow detected ***:  php terminated

### ISSUE THREAD

**Issue by mehmetbeyHZ** [beyondcode/laravel-websockets#751]
i have only a presence channel, 
```shell
ulimit -n 10000

/opt/plesk/php/7.4/bin/php /var/www/vhosts/fastfollowapp.com/artisan websockets:serve --port=6002
```

when app has 400 - 500 online users;
<img width="734" alt="Ekran Resmi 2021-04-29 17 14 20" src="https://user-images.githubusercontent.com/12996462/116648102-e94f5600-a984-11eb-839f-5aa4b0315017.png">

**Comment by DarkGhostHunter** [comment]
1.x ?

**Comment by mehmetbeyHZ** [comment]
> 1.x ?

1.9

**Comment by mehmetbeyHZ** [comment]
There is still no solution!

**Comment by DarkGhostHunter** [comment]
Does it happen with PHP 8X?

**Comment by mehmetbeyHZ** [comment]
PHP 7.4

**Comment by mpociot** [comment]
Which event loop are you using?

**Comment by mehmetbeyHZ** [comment]
> Which event loop are you using?
It was solved with version 2.0.0-beta.36 but now I have a new problem.
https://github.com/beyondcode/laravel-websockets/issues/755
